test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MultiClusterModeRunner.scala

```diff
@@ -0,0 +1,129 @@
+package com.typesafe.spark.test.mesos.framework.runners
+
+import java.io.File
+import java.net.URL
+
+import Utils._
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.spark.test.mesos.mesosstate.MesosCluster
+
+import org.apache.zookeeper.{ZKUtil, ZooKeeper, KeeperException}
+
+import scala.sys.process.Process
+
+object MultiClusterModeRunner {
```

Comment:
Ya I was having that debate myself, but I broke it off since I don't think we assumed zookeeper running before and now with mcm we do. 
I'm fine either way but like to get consensus if we can assume zookeeper is there, then I'm happy to just put them all in the mit